src/main.cpp

```diff
@@ -6162,35 +6202,23 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
 
         int count=0;
         size_t i;
```

Comment:
It would probably be safer if you moved the definition of `i` into the for loop, that avoids reusage later when its scope is not that useful anymore.